tools/webpack.assets.json

```diff
@@ -3,6 +3,10 @@
     "api": "./static/js/portico/api.js",
     "katex": ["./node_modules/katex/dist/katex.min.js"],
     "landing-page": "./static/js/portico/landing-page.js",
+    "stats": [
+              "./static/js/stats/stats.js",
+              "./node_modules/plotly.js/dist/plotly-basic.min.js"
```

Comment:
is there something we need to do for the fact that one of these is already minified, and the other isn't?